**********0************
**********100************
**********200************
**********300************
**********400************
**********500************
**********600************
**********700************
**********800************
**********900************
**********1000************
**********1100************
**********1200************
**********1300************
**********1400************
**********1500************
**********1600************
**********1700************
**********1800************
**********1900************
**********2000************
**********2100************
**********2200************
**********2300************
**********2400************
**********2500************
**********2600************
**********2700************
**********2800************
**********2900************
**********3000************
**********3100************
**********3200************
**********3300************
**********3400************
Traceback (most recent call last):
  File "vision_test.py", line 174, in <module>
    writer.writerow(vision_api(file_name))
  File "vision_test.py", line 74, in vision_api
    response.error.message))
Exception: Internal server error. Failed to process features.
For more info on error messages, check: https://cloud.google.com/apis/design/errors
